index.js

```
@@ -0,0 +1,11 @@
+import scrapeCategory, { fetchCategories } from './scraper.js';
+
+(async () => {
+  const categories = await fetchCategories();
+  for (const category of categories) {
+    console.log(`Scraping ${category.name}...`);
+    await scrapeCategory(category.url, category.name);
+    console.log(`Done with ${category.name}`);
+  }
+  process.exit();
+})();
```

Add error handling and improve process management.

The main execution lacks error handling, which could lead to unhandled promise rejections or silent failures during scraping operations.